pywbemtools/pywbemcli/_pywbem_server.py

```diff
@@ -519,7 +527,11 @@ def disconnect(self):
 
         Must be connected to the server when calling this method.
         """
-        # print("Debug: PywbemServer disconnect")
+        # TODO(PR900): Remove when disconnect fully tested
+        ctx = click.get_current_context()
+        if ctx.obj:
+            if ctx.obj.verbose:
+                print("Debug: PywbemServer disconnect")
```

Comment:
I wonder if we should leave the connect/disconnect messages in, in verbose mode.

Reply:
Actually I think we should, at least for this release.